libretro/libretro.cpp

```diff
@@ -602,62 +801,105 @@ bool retro_load_game(const struct retro_game_info *game)
    if (!rom_loaded && log_cb)
       log_cb(RETRO_LOG_ERROR, "[libretro]: Rom loading failed...\n");
 
+   struct retro_memory_map map={ memorydesc+MAX_MAPS-memorydesc_c, memorydesc_c };
+   if (rom_loaded) environ_cb(RETRO_ENVIRONMENT_SET_MEMORY_MAPS, &map);
```

Comment:
We don't support the memory mapping feedback, but I guess we can leave the callback and functions in if it makes it simpler for you.